### ISSUE TITLE
Issue with dgl v1.0.0

### ISSUE THREAD

**Issue by eereenah-fast** [awslabs/dgl-lifesci#208]
Hello!

I have updated `dgl` to `v1.0.0` and I am getting the following error when using code for **AttentiveFP** model:

```
File ".../anaconda3/envs/.../lib/python3.8/site-packages/dgllife/model/gnn/attentivefp.py", line 72, in forward
    g.update_all(fn.copy_edge('e', 'm'), fn.sum('m', 'c'))
AttributeError: module 'dgl.function' has no attribute 'copy_edge'
```
It seems that `fn.copy_edge()` has been replaced with `fn.copy_e()` in the newly released version of `dgl`.

**Comment by mufeili** [comment]
Thank you for the report. I confirm the issue and will check for all modules and fix them.

**Comment by mufeili** [comment]
This has been fixed in #210. Thank you for the report.
